frontend/packages/helm-plugin/src/components/__tests__/helm-release-mock-data.ts

```diff
@@ -224,9 +225,11 @@ export const mockChartEntries: HelmChartEntries = {
   'hazelcast-enterprise--redhat-helm-repo': mockRedhatHelmChartData,
 };
 
-export const mockReleaseResources: {
-  [key: string]: { data: K8sResourceKind };
-} = {
+export const mockReleaseResources: FirehoseResourcesResult<{
+  Deployment: K8sResourceKind;
+  StatefulSet: K8sResourceKind;
+  Pod: K8sResourceKind;
```

Comment:
Prefer `K8sResourceCommon` in most cases to `K8sResourceKind`. It has stricter types.

Reply:
replaced with `K8sResourceCommon`